### ISSUE TITLE
RFC: Improve documentation

### ISSUE THREAD

**Issue by jacobthemyth** [clintjhill/ember-parse-adapter#86]
It would be great to have documentation and guides for this library that are much more detailed. I would be willing to do the leg work on this, and have actually already started, but I have a couple questions I'd like feedback on.

1) Do they need to be versioned, or would a "single canonical site for docs" be sufficient?
2) Should they be in this repository, or a separate repo (a la https://github.com/emberjs/guides)? If there's only one monolithic library, I could see them being in a separate repo, but if there are many libraries as mentioned in #82, I'm not sure if a singe docs repo makes sense. On the other hand, having a single repo for docs would make it much easier to publish changes to the docs.

**Comment by joshfester** [comment]
My vote would be to keep all docs in in their separate repositories. But that's assuming we separate things as mentioned in #82, and that we move to a central organization as mentioned in #79

**Comment by mixonic** [comment]
@jacobthemyth the starting point should likely be updating the README to be more fleshed out. Additionally there are several wiki pages that could be updated or removed.

I strongly suggest that the most important thing is that the docs get written, and not that we need to be concerned with versioning them at this point. The classic programmer trap will drive you to want to code instead of write documentation :-) and that is the thing to avoid.

@jacobthemyth unless you have a resource who can organize a docs website better than the readme + wiki, I suggest we should start with those tools. Thoughts?

**Comment by clintjhill** [comment]
I would agree with README + Wiki. And I also agree versioning is unimportant at the moment.

**Comment by jacobthemyth** [comment]
@mixonic @clintjhill :+1: I'll open a pull request with some new docs for README asap.
